src/samples/wallet_demo.ts

```diff
@@ -49,15 +50,15 @@ doc.getElementById('publishAddresses').innerHTML = Object.keys(sampleAddressMap)
 
 // --- @crux/js-sdk integration --- //
 // defining cruxClientOptions
-const cruxClientOptions: ICruxClientOptions = {
+const cruxClientOptions: ICruxWalletClientOptions = {
```

Comment:
To validate backward compatibility can we have two versions of this file:
1. wallet_demo_backward_compatibility.ts ( old version of the file without any change.)
2. wallet_demo.ts (same as mentioned in this PR.

Reply:
https://github.com/cruxprotocol/js-sdk/pull/152/commits/afa87530e457604dc4ab44459261f141d1c73482